src/tests/JIT/Regression/JitBlue/Runtime_64883/Runtime_64883.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Optimize>True</Optimize>
```

Comment:
I wonder if this should be Pri-1; seems rare, and maybe the "load assembly" is relatively slow?

Reply:
Moved it to pri-1.